README.md

```diff
@@ -69,7 +69,21 @@ to the controller-gen CLI page in the [kubebuilder documentation](https://book.k
   ```sh
   kind create cluster --image kindest/node:v1.22.4
   ```
-
+  &nbsp; For more advanced tests, a 3 node cluster may be more suitable.
```

Comment:
How about we add this change to hack/kind/config.yaml since it is beneficial to have 3 node clusters to test HA.

Reply:
sounds good, done